webapp/src/store/contents.ts

```diff
@@ -30,7 +30,6 @@ const contentsSlice = createSlice({
                     for (let i = 0; i < state.contentsByCard[content.parentId].length; i++) {
                         if (state.contentsByCard[content.parentId][i].id === content.id) {
                             state.contentsByCard[content.parentId][i] = content
-                            return
```

Comment:
Probably a slightly more efficient approach is to replace it with a break. And also, I think we should avoid to push to the parent contentsByCard if it is already updated here.